Tribler/Test/Community/Multichain/test_multichain_database.py

```diff
@@ -166,8 +166,8 @@ def test_get_total_not_existing(self):
         # Act
         (result_up, result_down) = self.db.get_total(self.block2.public_key_requester)
         # Assert
-        self.assertEqual(-1, result_up)
-        self.assertEqual(-1, result_down)
+        self.assertEqual(0, result_up)
```

Comment:
I know, it seems like a wierd one. But... it is.. uhmmm.. inconvenient if the api reports the totals as -1 and -1. I suddenly had -2 for total contribution in the QT-gui. And there is no semantic reason to not default to (0, 0). Blocks with both zero up and zero down are invalid, and up and down cannot be negative, so after the first block we cannot have (0, 0). There are also changes to community.py _get_total and database.py get_total that result in 0,0 as the default. The test had to change too to reflect this change.

Similarly, I wonder if the last sequence number should read 0 instead of -1.

Also this change is inline with the upcomming true-halves for multichain.